frontend/app/src/components/status/notifications/MissingKeyNotification.vue

```diff
@@ -0,0 +1,21 @@
+<script setup lang="ts">
+import { type I18nParam } from '@rotki/common/lib/messages';
+
+defineProps<{
+  params: I18nParam;
+}>();
+</script>
+
+<template>
+  <i18n :path="params.message" :plural="params.choice" tag="div">
+    <template #service>
+      {{ params.props.service }}
+    </template>
+    <template #location>
+      {{ params.props.location }}
+    </template>
+    <template #url>
+      <base-external-link :href="params.props.url" :text="params.props.url" />
```

Comment:
@tewshi I changed this here to use the component that is designed to handle electron too